src/pages/Health.tsx

```diff
@@ -1,13 +1,141 @@
-import {Box, Heading} from '@chakra-ui/react';
+import {ExternalLinkIcon} from '@chakra-ui/icons';
+import {
+  Box,
+  Heading,
+  Text,
+  Link,
+  UnorderedList,
+  ListItem,
+  Image,
+  Flex,
+} from '@chakra-ui/react';
 import React from 'react';
 import {useTranslation} from 'react-i18next';
+import AqiTable from '../components/AqiTable';
 
 const Health: React.FC = () => {
   const {t} = useTranslation('health');
+
   return (
-    <Box>
-      <Heading>{t('heading')}</Heading>
-    </Box>
+    <Flex width="full" align="center" direction="column" padding={8}>
+      <Heading as="h1">{t('pageHeading')}</Heading>
+      <Flex
+        padding={8}
+        margin={8}
+        width="full"
+        borderWidth={1}
+        borderRadius={8}
+        boxShadow="lg"
+        direction="column"
+      >
+        <Heading>{t('pollution.heading')}</Heading>
+        <Text>{t('pollution.paragraph1')}</Text>
+        <Flex
+          padding={4}
+          margin={['2', 'auto', '2', 'auto']}
+          borderWidth={1}
+          borderRadius={8}
+          boxShadow="lg"
+          width={['100%', null, '525px', null]}
+        >
+          <Flex direction="column" align="center">
+            <Image
+              src={t('pollution.image.fileName')}
+              alt={t('pollution.image.caption')}
+              maxWidth={['95%', null, '500px', null]}
+            />
+            <Text>
+              {t('pollution.image.caption')}
+              <Link href={t('pollution.image.source.link')} isExternal>
+                {t('pollution.image.source.text')}
+                <ExternalLinkIcon />
+              </Link>
+            </Text>
+          </Flex>
+        </Flex>
+        <Text>
+          {t('pollution.paragraph2.part1')}{' '}
+          <Link href={t('references.health.link')} isExternal>
+            {t('pollution.paragraph2.link')} <ExternalLinkIcon />
+          </Link>
+          {t('pollution.paragraph2.part2')}
+          <UnorderedList>
```

Comment:
I believe this line is giving me an error in the console: `validateDOMNesting(...): <ul> cannot appear as a descendant of <p>.` Here, I think the wrapping `<Text>` tag is causing the issue. Instead, use a `<Box>`